phoneBook.js

```diff
@@ -17,29 +42,39 @@ module.exports.add = function add(name, phone, email) {
    Поиск ведется по всем полям.
 */
 module.exports.find = function find(query) {
-
-    // Ваша удивительная магия здесь
-
+    var findResult = findByQuery(query, phoneBook);
+    findResult.forEach(function (person) {
+        console.log(concatObject(person, ', '));
+    });
 };
 
 /*
    Функция удаления записи в телефонной книге.
 */
 module.exports.remove = function remove(query) {
-
-    // Ваша необьяснимая магия здесь
-
+    var findResult = findByQuery(query, phoneBook);
+    var countDeletedContacts = 0;
+    findResult.forEach(function (person) {
+        var index = phoneBook.indexOf(person);
+        phoneBook.splice(index, 1);
+        countDeletedContacts++;
+    });console.log('Удален %d контак(а,ов)', countDeletedContacts);
 };
 
 /*
    Функция импорта записей из файла (задача со звёздочкой!).
 */
 module.exports.importFromCsv = function importFromCsv(filename) {
-    var data = require('fs').readFileSync(filename, 'utf-8');
+    var countAddedContacts = 0;
+    var data = require('fs').readFileSync(filename, 'utf-8').split('\r\n');
 
-    // Ваша чёрная магия:
-    // - Разбираете записи из `data`
-    // - Добавляете каждую запись в книгу
+    for (var i = data.length - 1; i >= 0; i--) {
+        var entity = data[i].split(';');
+        if (module.exports.add(entity[0], entity[1], entity[2])) {
```

Comment:
@denchistyakov 
add(entity[0], entity[1], entity[2]);
        ^
ReferenceError: add is not defined
    at /home/alexeymm/Documents/yandex/javascript-tasks-2/phoneBook.js:69:9
    at Array.forEach (native)
    at Object.importFromCsv (/home/alexeymm/Documents/yandex/javascript-tasks-2/phoneBook.js:67:10)
    at Object.<anonymous> (/home/alexeymm/Documents/yandex/javascript-tasks-2/index.js:34:11)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)